packages/loader/container-loader/README.md

```diff
@@ -198,3 +199,10 @@ This information can be used by a host to build appropriate UX that allows user
 Note that when an active connection is in place, it's just a matter of time before changes will be flushed to storage unless there is some source of continuous local changes being generated that prevents container from ever being fully saved. But if there is no active connection, because the user is offline, for example, then a document may stay in a dirty state for very long time.
 
 `Container.isDirty` can be used to get current state of container.
+
```

Comment:
I'd suggest move this description to protocol-definitions package, as it's more appropriate there.